app/views/merchants/show.html.erb

```diff
@@ -4,6 +4,35 @@
   <section class="merchant-products__container">
     <%= link_to "Add product", new_product_path, class: "btn btn-primary" %>
 
+
+    <p><strong>Total Revenue: <%= @merchant.total_revenue%></strong></p>
+
+    <h3>Total Revenue Breakdown</h3>
+
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Order Status</th>
+          <th>Number of Orders</th>
+          <th>Total Revenue</th>
+        </tr>
+      </thead>
+
+      <tbody>
+
+      <% order_statuses = ["paid", "complete", "pending", "cancelled"] %>
+      <% order_statuses.each do |status| %>
+        <tr>
+          <td><%= status %></td>
+          <td> <%= @merchant.number_of_orders_for(status) %> </td>
+          <td><%= @merchant.revenue_for(status) %></td>
```

Comment:
See my comments above about combining this method with `total_revenue` and converting to currency ✨